tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -95,7 +95,7 @@ def test_that_reports_abuse_as_signed_in_user(self, mozwebqa):
 
         Assert.true(home_page.is_the_current_page)
         home_page.header.click_sign_in()
-        home_page.login(user="default")
+        home_page.login(user="new")
```

Comment:
Oh, I see here you're passing in `new` to indicate it's a new user, but that is still kind of confusing, as, theoretically, there could be a `new` entry in the credentials file. Also, the `login` method doesn't specifically look for a `new` user, so you could really pass anything here other than `default`. That seems like a loose API.